src/Microsoft.Diagnostics.Monitoring/Contracts/IDiagnosticServices.cs

```diff
@@ -21,9 +21,7 @@ public interface IDiagnosticServices : IDisposable
         Task<Stream> GetGcDump(int pid, CancellationToken token);
 
         //TODO We can most likely unify trace, cpu, and logs/metrics around one call with the appropriate config
```

Comment:
Do we want to remove this and the StartLogs call below?

Reply:
Thanks. I will remove the comment however the StartLogs method should remain since that is producing logs as newline delimited json objects. Though, I know that isn't obvious from the interface.